src/common/darktable.c

```diff
@@ -160,13 +164,13 @@ gboolean dt_supported_image(const gchar *filename)
 static void strip_semicolons_from_keymap(const char *path)
 {
   char pathtmp[PATH_MAX] = { 0 };
-  FILE *fin = fopen(path, "r");
+  FILE *fin = fopen(path, "rb");
```

Comment:
Yes, it's working. I am not sure if there is anything to fix from upstream. Actually I hope they don't change that as it might introduce bugs for us.